src/lua/database.c

```diff
@@ -206,6 +206,32 @@ static int database_numindex(lua_State *L)
   return 1;
 }
 
+static int database_get_image(lua_State *L)
+{
+  int img_id = luaL_checkinteger(L, -1);
+  if(img_id < 1)
+  {
+    return luaL_error(L, "incorrect image id in database");
+  }
+  sqlite3_stmt *stmt = NULL;
+  char query[1024];
+  snprintf(query, sizeof(query), "SELECT id FROM main.images WHERE id = %d LIMIT 1",
```

Comment:
Just to be sure, this returns img_id (the parameter) if image is found. So this routine is basically equivalent to a check if an image id exists in the database. Right ?

Reply:
It certainly looks that way :), but the return value from the query is the image object, so imgid != img_id

Reply:
I see, lua confused me :)